app/src/androidTest/java/com/github/se/cyrcle/di/mocks/MockParkingRepository.kt

```diff
@@ -0,0 +1,101 @@
+package com.github.se.cyrcle.di.mocks
+
+import com.github.se.cyrcle.model.parking.Parking
+import com.github.se.cyrcle.model.parking.ParkingRepository
+import com.github.se.cyrcle.model.parking.TestInstancesParking
+import com.mapbox.geojson.Point
+import com.mapbox.turf.TurfMeasurement
+import javax.inject.Inject
+
+class MockParkingRepository @Inject constructor() : ParkingRepository {
+  private var uid = 0
+  val parkings = mutableListOf<Parking>(TestInstancesParking.parking1)
```

Comment:
Everything in the mocks are private expect for this val

Reply:
Right, that was for debuging the E2E tests xD